picatic.io/app/components/header/header.module.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+module.exports = function(ngModule) {
+  ngModule.component('header',  {
+    template: require('./header.template.html'),
```

Comment:
I haven't seen this `require()` stuff here before. What's the purpose/benefit?

Reply:
This is just to let webpack know to include './header.template.html' in the bundle.js file.